content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -610,6 +610,31 @@ controller is tracking a Job using Pod finalizers by checking if the Job has the
 annotation `batch.kubernetes.io/job-tracking`. You should **not** manually add
 or remove this annotation from Jobs.
 
+### Mutable Scheduling Directives
```

Comment:
Better have this right after the "Suspending a Job" section, or even as a subsection within it.

Reply:
moved it right after the suspend section